docs/docs/glossary/build.md

```diff
@@ -9,7 +9,7 @@ Learn what _build_ means and how to set up a build process for your Gatsby proje
 
 _Build_ refers to the process of compiling your site. During a build, or at _build time_, your project gets transformed from component files to optimized HTML, CSS, and JavaScript files that you can [deploy](/docs/glossary#deploy) to your hosting provider.
 
-There are a few ways to create a build. You can build your site locally on your computer using the [Gatsby CLI](/docs/reference/gatsby-cli/#build), and then deploy changes to your [host](/docs/glossary#hosting). If you use [Gatsby Cloud](https://www.gatsbyjs.com/), you can take advantage of [Gatsby Builds](/blog/2020-01-27-announcing-gatsby-builds-and-reports/), a feature available with every Gatsby Cloud account. You can also use a [continuous deployment](/docs/glossary/continuous-deployment/) service such as [AWS Amplify](/docs/how-to/previews-deploys-hosting/deploying-to-aws-amplify/) or [Netlify](/docs/how-to/previews-deploys-hosting/deploying-to-netlify/).
+There are a few ways to create a build. You can build your site locally on your computer using the [Gatsby CLI](/docs/reference/gatsby-cli/#build), and then deploy changes to your [host](/docs/glossary#hosting). If you use [Gatsby Cloud](https://www.gatsbyjs.com/), you can take advantage of [Gatsby Builds](/blog/2020-01-27-announcing-gatsby-builds-and-reports/), a feature available with every Gatsby Cloud account. You can also use a [continuous deployment](/docs/glossary/continuous-deployment/) service such as [AWS Amplify](/docs/how-to/previews-deploys-hosting/deploying-to-aws-amplify/), [Netlify](/docs/how-to/previews-deploys-hosting/deploying-to-netlify/), [21YunBox](/docs/how-to/previews-deploys-hosting/deploying-to-21yunbox/).
```

Comment:
Please revert that change, it's out of scope of this PR and the paragraph is purposefully only listing those two options. Thanks!

Reply:
done!